ray-operator/controllers/ray/utils/util.go

```diff
@@ -689,6 +677,11 @@ func IsGCSFaultToleranceEnabled(spec *rayv1.RayClusterSpec, annotations map[stri
 	return (ok && strings.ToLower(v) == "true") || spec.GcsFaultToleranceOptions != nil
 }
 
+// IsAuthEnabled returns whether Ray auth is enabled.
+func IsAuthEnabled(spec *rayv1.RayClusterSpec) bool {
```

Comment:
```suggestion
func IsTokenAuthEnabled(spec *rayv1.RayClusterSpec) bool {
```

Reply:
done

Reply:
Weird. I am still seeing it remains the old IsAuthEnabled.

Reply:
sorry, I probably forgot to push the latest commits